docs/releases/2.13.rst

```diff
@@ -95,3 +102,16 @@ Setting menu items now use SVG icons by default. For sites reusing built-in Wagt
         return SettingMenuItem(CustomSetting, icon='custom-cog')
         # After:
         return SettingMenuItem(CustomSetting, icon='', classnames='icon icon-custom-cog')
+
+
+CommentPanel
+~~~~~~~~~~~~
+
+``Page.content_panels`` now includes `CommentPanel`, which is used to save and load comments. If you are overriding page edit handlers and
+want to use the new commenting system, your list of edit handlers should be updated to include ``CommentPanel``.
```

Comment:
Could do with a code example here, to show what the import and declaration for `CommentPanel` look like, and clarify that you only need it if your `content_panels` definition _doesn't_ incorporate `Page.content_panels`.

Reply:
Thanks, will add some code.

Reply:
Added now